pkg/disttask/framework/integrationtests/framework_pause_and_resume_test.go

```diff
@@ -46,11 +48,20 @@ func TestFrameworkPauseAndResume(t *testing.T) {
 
 	testutil.RegisterTaskMeta(t, c.MockCtrl, testutil.GetMockBasicSchedulerExt(c.MockCtrl), c.TestContext, nil)
 	// 1. schedule and pause one running task.
-	testkit.EnableFailPoint(t, "github.com/pingcap/tidb/pkg/disttask/framework/scheduler/pauseTaskAfterRefreshTask", "2*return(true)")
+	cnt := 0
+	scheduler.OnTaskRefreshed = func(ctx context.Context, taskMgr scheduler.TaskManager, task *proto.Task) {
+		if cnt < 1 && task.State == proto.TaskStateRunning {
+			_, err := taskMgr.PauseTask(ctx, task.Key)
+			if err != nil {
+				logutil.BgLogger().Error("pause task failed", zap.Error(err))
+			}
+			task.State = proto.TaskStatePausing
```

Comment:
although copied from prevous failpoint, we shouldn't change task state here, we're testing that state changed by others after refresh only.